### PR TITLE
Adding an OSGi Console for the FSClassLoader

### PR DESCRIPTION
**Pull request by klcodanr**
Related to: https://issues.apache.org/jira/browse/SLING-4792
screen shot 2015-06-09 at 1 22 05 pm
screen shot 2015-06-09 at 1 22 15 pm

### CODE REVIEW

**Comment by rombert** [comment]
Looks good to me! A couple of inline notes regarding the implementation will follow

File: bundles/commons/fsclassloader/pom.xml [hunk 1]
```diff
@@ -99,5 +99,29 @@
             <version>1.3.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.tomcat</groupId>
```
Comment (rombert) [inline bundles/commons/fsclassloader/pom.xml]: Couldn't this be javax.servlet?

File: bundles/commons/fsclassloader/src/main/java/org/apache/sling/commons/fsclassloader/impl/FSClassLoaderWebConsole.java [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+	 * @param componentContext
+	 * @throws MalformedURLException
+	 */
+	@Activate
+	protected void activate(final ComponentContext componentContext)
+			throws MalformedURLException {
+		// get the file root
+		root = new File(componentContext.getBundleContext().getDataFile(""),
+				"classes");
+		root.mkdirs();
```
Comment (rombert) [inline bundles/commons/fsclassloader/src/main/java/org/apache/sling/commons/fsclassloader/impl/FSClassLoaderWebConsole.java]: Why do we need to create directories?

**Comment by klcodanr** [comment]
Thanks @rombert!  I made the recommended changes.

**Pull request closed without merging**